modules/cactus-web/src/DateInput/DateInput.tsx

```diff
@@ -1184,19 +1296,54 @@ class DateInputBase extends Component<DateInputProps, DateInputState> {
       )
     } else if (key === ' ' || key === 'Enter') {
       wasHandled = true
-      if (currentTarget.dataset.token === 'YYYY') {
-        this.togglePortalView()
-      } else {
-        // @ts-ignore
-        currentTarget.parentElement.nextSibling.firstChild.focus()
-      }
+      this.togglePortalView()
     }
     if (wasHandled) {
       event.preventDefault()
       event.stopPropagation()
     }
   }
 
+  private handleLeftMonthArrowClick = (event: React.MouseEvent<HTMLButtonElement>): void => {
+    const { month: focusMonth, year: focusYear } = this._getMonthData()
+    this.setState(
+      (state): Pick<DateInputState, 'focusDay' | 'value' | 'transitioning'> => {
+        const { value } = state
+        const pd = PartialDate.from(value, 'YYYY-MM-dd')
+        pd['setMonth'](focusMonth === 0 ? 11 : focusMonth - 1)
```

Comment:
Doesn't matter too much, but I think you can do the following, right?

```suggestion
        pd.setMonth(focusMonth === 0 ? 11 : focusMonth - 1)
```